src/bar_chart.py

```diff
@@ -0,0 +1,92 @@
+import pandas as pd
+import altair as alt
+
+def bar_chart_gen(data, genrelist = []):
```

Comment:
```suggestion
def bar_chart_gen(data, genrelist=[]):
```

Reply:
The `genrelist=[]` parameter will be deleted since we are filtering within app.py itself.